app/views/admin/intervention_type_groups/show.html.erb

```diff
@@ -1,5 +1,5 @@
 <h1>Intervention Category</h1>
-<p><strong>Title:</strong> <%= @intervention_type_group.title %></p>
+<p><strong>Title:</strong> <%= @intervention_type_group.name %></p>
```

Comment:
Wondering why this changed?

Reply:
sorry @urbanwide  I should have left a comment explaining that.  it's a bug fix - no `title` on InterventionTypeGroups 
https://rollbar.com/energysparks/EnergySparksProduction/items/1642/
(prob should have gone in a different pr but i only noticed the issue when testing this one)

Reply:
Ahh cool, makes sense :-) Thanks for explaining